File: EulerProject/0xx/00x/007/main.py
prime = [2,3]
i = 4
while len(prime)<10001:
    s = True
    for j in prime:
        if i%j == 0:
            s = False
            break
    if s:
        prime.append(i)
    i = i + 1
print(prime[-1])

File: EulerProject/0xx/00x/007/main.cs
using System;
using System.Collections.Generic;

public class Prime {
    public static void Main() {
        var primeTable = new List<ulong>();
        ulong i = 2;
        while (primeTable.Count < 1e4 + 1) {
            foreach (var elem in primeTable)
                if (i % elem == 0)
                    goto loop;
            primeTable.Add(i);
        loop:
            i++;
        }
        Console.WriteLine(primeTable[primeTable.Count-1]);
    }
}